src/test/features/claim/routes/check-and-send.ts

```diff
@@ -136,6 +356,186 @@ describe('Claim issue: check and send page', () => {
           .send({ signed: 'true' })
           .expect(res => expect(res).to.be.redirect.toLocation(ClaimPaths.startPaymentReceiver.uri))
       })
+
+      it('Should validate that a claim made by individual against soleTrader and redirect to payment page', async () => {
```

Comment:
repetition of post queries ... redundant

Reply:
Removed